projects/aca-content/src/lib/components/details/details.component.ts

```diff
@@ -99,6 +99,22 @@ export class DetailsComponent extends PageComponent implements OnInit, OnDestroy
       .subscribe((aspectActions) => {
         this.aspectActions = aspectActions;
       });
+    this.infoDrawerOpened$
+      .pipe(
+        filter((opened) => !opened),
+        first(),
+        takeUntil(
+          merge(
+            this.onDestroy$,
+            this.router.events.pipe(
+              filter((event) => event instanceof NavigationStart),
```

Comment:
Same points here for `filter` and `takeUntil`

Reply:
same answer like above